docs/content/index.mdx

```diff
@@ -58,6 +58,22 @@ from aws_lambda_powertools.logging.logger import set_package_logger
 set_package_logger()
 ```
 
+## Lambda Layer
+
+Powertools is also available as a layer and is distributed via the [Serverless Application Repository](https://docs.aws.amazon.com/serverlessrepo/latest/devguide/what-is-serverlessrepo.html).
+Create the layer via API with:
+
+```bash:title=bash-script
+aws serverlessrepo create-cloud-formation-change-set --application-id arn:aws:serverlessrepo:eu-west-1:057560766410:applications/aws-lambda-powertools-python-layer --stack-name YOUR_STACK_NAME --capabilities CAPABILITY_NAMED_IAM
+CHANGE_SET_ID=$(aws cloudformation list-change-sets --stack-name YOUR_STACK_NAME --query 'Summaries[*].ChangeSetId' --output text)
+aws cloudformation wait change-set-create-complete --change-set-name $CHANGE_SET_ID
+aws cloudformation execute-change-set --change-set-name $CHANGE_SET_ID
+```
+
+this will create a CloudFormation stack with the powertools layer in a specific region where you run this commands.
+Keep in mind that layers are regional resources and you need to create the layer in every region where you need it.
+Alternatively, you can deploy the layer from the AWS Console by navigating to Serverless Application Repository and search for `aws-lambda-powertools-python-layer` and follow the deploy steps from there.
```

Comment:
```suggestion
SAR App | ARN
------------------------------------------------- | ---------------------------------------------------------------------------------
[aws-lambda-powertools-python-layer](https://serverlessrepo.aws.amazon.com/applications/eu-west-1/057560766410/aws-lambda-powertools-python-layer) | arn:aws:serverlessrepo:eu-west-1:057560766410:applications/aws-lambda-powertools-python-layer

If using SAM, you can include this SAR App as part of your shared Layers stack, and lock to a specific semantic version. Once deployed, it'll be available across the account this is deployed to.

```yaml
  AwsLambdaPowertoolsPythonLayer:
    Type: AWS::Serverless::Application
    Properties:
      Location:
        ApplicationId: arn:aws:serverlessrepo:eu-west-1:057560766410:applications/aws-lambda-powertools-python-layer
        SemanticVersion: 1.3.0 # change to latest semantic version available in SAR
```

Customers typically use SAR within their SAM stack, or the Console, but rarely using the CLI as it's quite a lot of steps. Added a table as well as a snippet of this SAR App